sdk/eventhub/azure-eventhub/samples/sync_samples/send.py

```diff
@@ -96,4 +96,4 @@ def send_event_data_list(producer):
     send_event_data_batch_with_properties(producer)
     send_event_data_list(producer)
 
-print("Send messages in {} seconds.".format(time.time() - start_time))
+print("Send messages in {} seconds.".format(time.time() - start_time))
```

Comment:
looks like no changes were made to this sample, so you can probably revert it? I think the CI pylint might fail since this file doesn't have a newline at the end.